scrapers/fifty_a/run.sh

```diff
@@ -5,3 +5,6 @@ scrapy crawl officer -O officers.jsonl
 scrapy crawl command -O commands.jsonl
```

Comment:
Should we create variables for `date +"%Y-%m-%dT%H-%M-%S"` and `date +"%Y-%m-%d"` and re-use those? Looks like we will be using them over and over.

Also, it might be better to move this script one level above since it is not fifty_a specific anymore